modules/pam_access/pam_access.c

```diff
@@ -605,7 +606,30 @@ netgroup_match (pam_handle_t *pamh, const char *netgroup,
   return retval;
 }
 
-/* user_match - match a username against one token */
+/* user_name_or_uid_match - match a username or user uid against one token */
+static int
+user_name_or_uid_match (pam_handle_t *pamh, const char *tok, const struct login_info *item)
+{
+    /* check for exact match of username */
+    if(string_match (pamh, tok, item->user->pw_name, item->debug) != NO) {
+        return YES;
+    }
```

Comment:
`string_match` can return other values besides `YES` and `NO`, so this would be a change of behavior.